### ISSUE TITLE
I've try and it always return hello word, could you please help, Thanks

### ISSUE THREAD

**Issue by huynhlexuan** [localstack-samples/lambda-sam-authorizer#2]
It is never call to authorizer
<img width="852" alt="image" src="https://github.com/user-attachments/assets/51fdf75e-9a07-41e6-b5fa-d81b2688d014">

**Comment by markdynamo6** [comment]
Yep.  Doesn't appear to work for me either. The authoriser is never called. Please advise.
